wandb/sdk/internal/file_pusher.py

```diff
@@ -157,14 +133,6 @@ def store_manifest_files(self, manifest, artifact_id, save_fn):
         event = step_checksum.RequestStoreManifestFiles(manifest, artifact_id, save_fn)
         self._incoming_queue.put(event)
 
-    def named_temp_file(self, mode="w+b"):
-        # get a named temp file that the file pusher with hold a reference to so it
-        # doesn't get gc'd. Obviously, we shouldn't do this very much :). It's currently
-        # used for artifact metadata.
-        f = builtin_tempfile.NamedTemporaryFile(mode=mode, delete=False)
-        self._temp_file_refs.append(f)
-        return f
-
```

Comment:
I don't think this method is used anywhere.

Reply:
looks like it!